types/types.proto

```diff
@@ -162,9 +162,9 @@ message ResponseEndBlock{
 
 message Header {
   string chain_id = 1;
-  uint64 height = 2;
-  uint64 time = 3;
-  uint64 num_txs = 4;
+  int64 height = 2;
+  int64 time = 3;
+  int64 num_txs = 4;
```

Comment:
same here. or we decided to completely get rid of uint64?

Reply:
You're probably right, but I think best to just be consistent, no?